tutorials/wpa-enterprise-eap-ttls-pap.rst

```diff
@@ -219,75 +217,71 @@ need to update these fields to reflect your configuration:
 .. code-block:: json
 
     {
-        "interfaces": [
-            {
-                "wireless": {
-                    "network": [
-                        "lan"
-                    ],
-                    "mode": "access_point",
-                    "radio": "radio0",
-                    "ack_distance": 0,
-                    "rts_threshold": 0,
-                    "frag_threshold": 0,
-                    "ssid": "OpenWISP",
-                    "hidden": false,
-                    "wds": false,
-                    "encryption": {
-                        "protocol": "wpa2_enterprise",
-                        "key": "testing123",
-                        "disabled": false,
-                        "cipher": "auto",
-                        "ieee80211w": "0",
-                        "server": "10.8.0.1",
-                        "port": 1822,
-                        "acct_server": "10.8.0.1",
-                        "acct_server_port": 1823
-                    },
-                    "wmm": true,
-                    "isolate": false,
-                    "ieee80211r": false,
-                    "reassociation_deadline": 1000,
-                    "ft_psk_generate_local": false,
-                    "ft_over_ds": true,
-                    "rsn_preauth": false,
-                    "macfilter": "disable",
-                    "maclist": []
-                },
-                "type": "wireless",
-                "name": "wlan0",
-                "mtu": 1500,
-                "disabled": false,
-                "network": "",
-                "mac": "",
-                "autostart": true,
-                "addresses": []
+        "interfaces": [{
```

Comment:
Did you update the fixture here accordingly?
https://github.com/openwisp/ansible-openwisp-cloud/pull/74

Reply:
Yes, I have.